Neos.Neos/Documentation/Appendixes/ReleaseNotes/430.rst

```diff
@@ -0,0 +1,141 @@
+While this new view (implemented as a tab) may be useful already, it is neither especially polished nor does it provide additional features. 
+Therefore it is disabled by a feature switch for now. See the ``Settings.yaml`` of ``Neos.Media.Browser`` for details on enabling the tab.
+
+Add fallback from ``itemRenderer`` to ``content`` for ``Neos.Fusion:Map`` and derived prototypes
+================================================================================================
+
+This allows to declare the itemRenderers in `AFX` without `@children` annotation and avoids a
+common error in afx code.
+
+Affected prototypes:
+-  `Neos.Fusion:Map`
+-  `Neos.Fusion:RawCollection`
+-  `Neos.Fusion:Loop`
+-  `Neos.Fusion:Collection`
+
+Native 404 error rendering that can be controlled via Fusion
+============================================================
+This change adds the default Fusion path `error` that is used to render status messages
+for the status codes 403, 404 and 410.
+
+The matcher receives the context values `exception`, `renderingOptions`, `statusCode`,
+ `statusMessage` and `referenceCode` and will by default render the previous template.
+
+By extending the `error` case you can add custom 404 rendering like in the example below.
+
+```
+	#
+	# Extend error matcher to render the document with uriPathSegment `notfound`
+	# for exceptions with 4xx status code
+	#
+	error {
+		@context.notfoundDocument = ${q(site).children('[instanceof Neos.Neos:Document]').filter('[uriPathSegment="notfound"]').get(0)}
+
+		4xx {
+			@position = 'start'
+			condition = ${statusCode >= 400 && statusCode < 500 && notfoundDocument}
+			renderer = Neos.Fusion:Renderer {
+				@context.node = ${notfoundDocument}
+				renderPath = '/root'
+			}
+		}
+	}
+```
+Set crop image adjustment by aspect ratio
+=========================================
+This change introduces a new property "aspectRatio" for the crop image
+adjustment. It allows users to set a cropping area simply by providing
+an aspect ratio, instead of x, y, width and height.
+
+If an aspect ratio is specified, the x, y, width, and height parameters
+are automatically deactivated. Likewise, if x, y, width or height are
+specified, a potentially defined aspect ratio value will be reset.
+
+
+Add ``format`` option for image-prototypes, viewHelpers and presets
+===================================================================
+The option ``format`` is added to imagePresets, viewHelpers and FusionPrototypes.
+The format is passed as string "jpg", "jpeg", "gif", "png", "wbmp", "xbm", "webp" and "bmp" are supported. 
+If no format is given the crops will use the format of the original image.
+
+This allows to to enforce rendering of crops in jpeg or png but also adds support for rendering
+of webp-images as alternate sources.
+
+Additionally this pr adds the ConvertImageThumbnailGenerator that allows to specify images that should be converted for 
```

Comment:
This part will hopefully be replaced with https://github.com/neos/neos-development-collection/pull/2431